bayesian/v2/api_v2.py

```diff
@@ -180,6 +188,38 @@ def get(ecosystem, package, version):
             raise HTTPError(404, msg)
 
 
+class StackAnalysesPost(Resource):
+    """Implementation of /stack-analyses POST REST API call."""
```

Comment:
class StackAnalysesApi:
def get(): ... 
def post(): ...

Why are we separating on Classes..?

Reply:
This will corrected.